core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineMysql.java

```diff
@@ -448,7 +449,19 @@ public TimestampTZColumn parseTimestampTZColumn(ResultSet resultSet, String colu
   }
 
   @Override
-  public Map<String, String> getConnectionProperties() {
+  public Map<String, String> getConnectionProperties(JdbcConfig config) {
+    if (config.getDatabaseConfig().getScanFetchSize() == Integer.MIN_VALUE) {
+      // If the scan fetch size is set to Integer.MIN_VALUE, use the streaming mode.
```

Comment:
[question] Is this behavior only for MySQL?

Reply:
Yes.

Reply:
Don't need to add an integration test using `scalar.db.scan_fetch_size = Integer.MIN_VALUE` just in case ?

Reply:
Okay, let me do that.